llama-index-legacy/llama_index/legacy/readers/file/tabular_reader.py

```diff
@@ -46,10 +46,15 @@ def load_data(
             csv_reader = csv.reader(fp)
             for row in csv_reader:
                 text_list.append(", ".join(row))
+
+        metadata = {"filename": file.name, "extension": file.suffix}
```

Comment:
don't think you need to change the legacy code. can just stash the changes here

Reply:
Yea, will merge once this change is removed

Reply:
Fixed!